fluent/toInterfaceValue.go

```diff
@@ -0,0 +1,67 @@
+package fluent
+
+import (
+	"github.com/ipld/go-ipld-prime/datamodel"
+)
+
+func ToInterfaceValue(node datamodel.Node) (interface{}, error) {
```

Comment:
After having slept on my suggestion of ToInterfaceValue, I wonder if just ToInterface would be enough. All functions that return `(T, err)` already return a value.